src/eval_utils/types.ts

```diff
@@ -90,8 +90,10 @@ export interface Evaluator extends Identifiers {
     returnType?: EvaluatorReturnTypeEnum;
     /**The threshold to check the Evaluator against. If the aggregate value of the Evaluator is below this threshold, the check will fail.*/
     threshold?: number;
-    callable: Function;
-    argsType: EvaluatorArgumentsType;
+    /**The function to run on the logs to produce the judgment - only required for local Evaluators.*/
+    callable?: Function;
+    /**The type of arguments the Evaluator expects - only required for local Evaluators.*/
+    argsType?: EvaluatorArgumentsType;
```

Comment:
I suspect we should/could do some stronger typing here where we can statically check if a passed evaluator is either a remote evaluator (path/id) or local one.

Reply:
true, evaluator could be a union where you either provide a path or {callable, argsType, returnType}

Reply:
<img width="1045" alt="image" src="https://github.com/user-attachments/assets/f218f34f-b8f9-4709-9f6a-b7cc59e2489d" />